app/models/item.rb

```diff
@@ -0,0 +1,7 @@
+class Item < ActiveRecord::Base
+  belongs_to :user
+
+  def create
```

Comment:
This overrides the inherited `ActiveRecord::Base.create` method, which is probably not something you want to do.  Try `Item.create(..)` in the console and you will likely find it's broken.